documentation/ci-cd-strategy.md

```diff
@@ -0,0 +1,59 @@
+# Find, API and publish CI/CD
+
+Today the deployment process is somewhat manual, unreliable and slow. This creates frustration for the devs and it takes up a lot of their time. As we are moving to GOV.UK paas and Github Actions, we can take the opportunity to move to a continuous deployment model, but with the right controls in place. Here is the proposal.
+
+### Spaces
+We have 3 spaces in paas: bat-qa, bat-staging, bat-prod.
+* The managers (tech leads etc) are SpaceManagers so they can add/remove SpaceDevelopers
+* Developers are SpaceDevelopers in bat-qa. They can be made temporarily SpaceDeveloper in staging and prod by a SpaceManager
+
+### Environments
+All the permanent environments are on paas, with the same names as today: qa, staging, prod.
```

Comment:
We translated the existing Azure environments such QA, Staging and Prod to PaaS.

Reply:
Done, thanks